extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -630,13 +629,11 @@ export class SystemLayer {
    */
   onPageHasElementsWithPlaybackStateUpdate_(pageHasElementsWithPlayback) {
     this.vsync_.mutate(() => {
-      toArray(
-        this.getShadowRoot().querySelectorAll(
-          '.i-amphtml-paused-display button'
-        )
-      ).forEach((button) => {
-        button.disabled = !pageHasElementsWithPlayback;
-      });
+      this.getShadowRoot()
```

Comment:
We need to keep the `toArray()` for browser compatibility. Some browsers don't have `NodeList.forEach` in their prototype.